### ISSUE TITLE
YACL 3.0 crashes Minecraft 1.20

### ISSUE THREAD

**Issue by OpenBagTwo** [isXander/YetAnotherConfigLib#74]
Minecraft version: 1.20
Loader: Fabric 0.14.21
YACL version: 3.0.0+1.20, downloaded from Modrinth
Other mods present: Fabric API 0.83.0+1.20
System: Linux x86_64

### To reproduce
1. Load Minecraft
2. Load any world

Minecraft crashes completely

### Error Message
```
java.util.ServiceConfigurationError: javax.imageio.spi.ImageReaderSpi: Provider com.twelvemonkeys.imageio.plugins.webp.WebPImageReaderSpi could not be instantiated
```

[Full crash report](https://github.com/isXander/YetAnotherConfigLib/files/11682067/crash-2023-06-07_17.19.31-server.txt)

Please let me know if I can be of any assistance.

**Comment by OpenBagTwo** [comment]
FYI: after reverting e8e16fd97147dd1693189b8b3fef9a6883dec683 and rebuilding the mod, I was able to launch into a Minecraft world and use a mod that depends on this library.

**Comment by cadenkriese** [comment]
I get crashes on 1.19.4 when opening the mod menu or multiplayer menu with 3.0 as well. Beta 2 is fine though

**Comment by minhperry** [comment]
YACL 3.0-1.20 also caused Xaero's Map to crash ([log here](https://mclo.gs/I8Bmgys))

**Comment by Qendolin** [comment]
This also causes an error when fabric detects a missing dependency:

```
[17:26:14] [main/ERROR] (FabricLoader) Incompatible mod set!
 net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Install yet_another_config_lib, any version.
Unmet dependency listing:
	 - Mod 'Better Clouds' (betterclouds) ${version} requires any version of yet_another_config_lib, which is missing!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:192) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) ~[?:?]
[17:26:15] [main/WARN] (FabricLoader) Failed to open the error gui!
 java.lang.reflect.InvocationTargetException: null
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1369) ~[?:?]
	at java.awt.EventQueue.invokeAndWait(EventQueue.java:1344) ~[?:?]
	at javax.swing.SwingUtilities.invokeAndWait(SwingUtilities.java:1480) ~[?:?]
	at net.fabricmc.loader.impl.gui.FabricMainWindow.open0(FabricMainWindow.java:94) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.gui.FabricMainWindow.open(FabricMainWindow.java:88) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.gui.FabricGuiEntry.open(FabricGuiEntry.java:51) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.gui.FabricGuiEntry.displayError(FabricGuiEntry.java:141) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.gui.FabricGuiEntry.displayError(FabricGuiEntry.java:108) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.FabricLauncherBase.handleFormattedException(FabricLauncherBase.java:84) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:76) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) ~[?:?]
Caused by: java.util.ServiceConfigurationError: javax.imageio.spi.ImageReaderSpi: Provider com.twelvemonkeys.imageio.plugins.webp.WebPImageReaderSpi not found
	at java.util.ServiceLoader.fail(ServiceLoader.java:593) ~[?:?]
	at java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1219) ~[?:?]
	at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228) ~[?:?]
	at java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273) ~[?:?]
	at java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309) ~[?:?]
	at java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393) ~[?:?]
	at javax.imageio.spi.IIORegistry.registerApplicationClasspathSpis(IIORegistry.java:185) ~[?:?]
	at javax.imageio.spi.IIORegistry.<init>(IIORegistry.java:113) ~[?:?]
	at javax.imageio.spi.IIORegistry.getDefaultInstance(IIORegistry.java:134) ~[?:?]
	at javax.imageio.ImageIO.<clinit>(ImageIO.java:66) ~[?:?]
	at net.fabricmc.loader.impl.gui.FabricMainWindow.loadImage(FabricMainWindow.java:225) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.gui.FabricMainWindow.createUi(FabricMainWindow.java:109) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.gui.FabricMainWindow.lambda$open0$0(FabricMainWindow.java:95) ~[fabric-loader-0.14.21.jar:?]
	at java.awt.event.InvocationEvent.dispatch$$$capture(InvocationEvent.java:308) ~[?:?]
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java) ~[?:?]
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:771) ~[?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:722) ~[?:?]
	at java.awt.EventQueue$4.run(EventQueue.java:716) ~[?:?]
	at java.security.AccessController.doPrivileged(AccessController.java:399) ~[?:?]
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86) ~[?:?]
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:741) ~[?:?]
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203) ~[?:?]
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124) ~[?:?]
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113) ~[?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109) ~[?:?]
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) ~[?:?]
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:90) ~[?:?]
```

**Comment by Excederus** [comment]
Just want to let you guys know that this is fixed with v3.0.1

**Comment by Qendolin** [comment]
3.0.1+1.20 isn't on maven yet

**Comment by TH3N3WN00B** [comment]
I get crashes with latest YACL (3.0.1 - 1.19.4) and latest zoomify 2.10.0

https://paste.ee/p/NjQvq

**Comment by isXander** [comment]
> I get crashes with latest YACL (3.0.1 - 1.19.4) and latest zoomify 2.10.0
> 
> [paste.ee/p/NjQvq](https://paste.ee/p/NjQvq)

This is a zoomify issue, and has been dealt with on that repo.

**Comment by isXander** [comment]
Fixed on 3.0.1

**Comment by HardCoreFan** [comment]
I also have the same problem.

Minecraft Version: 1.21.3
Mod loader: Fabric
Mod loader version: 0.16.9
Java version: 23.0.1
